contracts/HeapOrdering.sol

```diff
@@ -123,13 +123,13 @@ library HeapOrdering {
     /// @param _heap The heap to modify.
     /// @param _rank The rank of the account to move.
     function shiftUp(HeapArray storage _heap, uint256 _rank) private {
-        Account memory initialAccount = getAccount(_heap, _rank);
-        uint256 initialValue = initialAccount.value;
-        while (_rank > 1 && initialValue > getAccount(_heap, _rank >> 1).value) {
+        Account memory accountToShift = getAccount(_heap, _rank);
+        uint256 valueToShift = accountToShift.value;
```

Comment:
This variable saves 5 gas on average for `updateHeap` (just tried). We don't have to keep it (but we can)

Reply:
let's keep I'de say